tester/tester.go

```diff
@@ -383,6 +383,26 @@ func (tt *Tester) waitForClients() {
 	logger.Printf("waiting for consumers done")
 }
 
+// GetTableKeys returns a Table's keys.
+func (tt *Tester) GetTableKeys(table goka.Table) []string {
+	tt.mStorages.Lock()
+	defer tt.mStorages.Unlock()
+
+	keys := make([]string, 0)
+	topic := string(table)
+	st, exists := tt.storages[topic]
+	if !exists {
+		panic(fmt.Errorf("topic %s does not exist", topic))
```

Comment:
you don't have to panic, you can actually use the `testing.T` that is added when creating the Tester: `tt.t.Fatalf(...)`